src/main/java/briscoe/cleancode/VehicleFormatter.java

```diff
@@ -17,28 +29,22 @@ public String toMarkdown() {
 	}
 
 	private String render(final String template) {
-		return String.format(template, 
-				vehicle.getVin(), 
-				vehicle.isBlacklisted(), 
-				vehicle.getMake(), 
+		return String.format(template,
+				vehicle.getVin(),
+				vehicle.isBlacklisted(),
+				vehicle.getMake(),
 				vehicle.getModel(),
-				vehicle.getYear(), 
-				vehicle.getColor(), 
+				vehicle.getYear(),
+				vehicle.getColor(),
 				vehicle.getMiles());
 	}
 
 	private String htmlTemplate() {
-		final StringBuilder html = new StringBuilder();
-		html.append("<h1>Vehicle Details</h1>\n");
-		html.append("<table>\n");
-		html.append("<tr><th>VIN</th><th>Blacklisted</th><th>Make</th><th>Model</th><th>Year</th><th>color</th><th>Miles</th></tr>\n");
-		html.append("<tr>\n");
-		html.append("<td>%1$s</td><td>%2$s</td><td>%3$s</td><td>%4$s</td><td>%5$s</td><td>%6$s</td><td>%7$s</td>\n");
-		html.append("</tr>\n");
-		html.append("</table>");
+		final String html = HTML_TEMPLATE;
+
 		return html.toString();
 	}
-	
+
 	private String markdownTemplate() {
```

Comment:
You should use the new `MARKDOWN_TEMPLATE` variable in this method too. Right now it's duplicated still.

Reply:
I thought I had, it's done now